lib/livebook_web/live/session_live/cell_component.ex

```diff
@@ -191,14 +191,23 @@ defmodule LivebookWeb.SessionLive.CellComponent do
           <div class="input-label">
             <%= @cell_view.name %>
           </div>
-          <input type="<%= if(@cell_view.input_type == :password, do: "password", else: "text") %>"
-            data-element="input"
-            class="input <%= if(@cell_view.error, do: "input--error") %>"
-            name="value"
-            value="<%= @cell_view.value %>"
-            spellcheck="false"
-            autocomplete="off"
-            tabindex="-1" />
+          <%= if (@cell_view.input_type == :textarea) do %>
+            <textarea
+              data-element="input"
+              class="input <%= if(@cell_view.error, do: "input--error") %>"
+              name="value"
+              spellcheck="false"
+              tabindex="-1"/><%= @cell_view.value %></textarea>
```

Comment:
We need to include a newline after the textarea. This is because the HTML spec says new lines after tags must be ignored and all major browser implementations do that. Otherwise if the user writes a newline, it will be ignored. We should also remove the self-closing from the textarea.


```suggestion
              tabindex="-1">
<%= @cell_view.value %></textarea>
```

Reply:
This change makes the formatting test not pass, if you give the formatting spaces, this space is applied in the textarea.

Is that okay?

Reply:
I think you just need to indent the line to the same level as `"""`, it won't add any leading white space.